src/libs/actions/SignInRedirect.js

```diff
@@ -43,8 +46,10 @@ function clearStorageAndRedirect(errorMessage) {
                 Onyx.set(ONYXKEYS.NETWORK, {isOffline});
             }
 
-            // `Onyx.clear` reinitialize the Onyx instance with initial values so use `Onyx.merge` instead of `Onyx.set`.
-            Onyx.merge(ONYXKEYS.SESSION, {error: errorMessage});
+            // `Onyx.clear` reinitialize the Onyx instance with initial values so use `Onyx.merge` instead of `Onyx.set`
+            if (errorMessage) {
+                Onyx.merge(ONYXKEYS.SESSION, {errors: {[DateUtils.getMicroseconds()]: Localize.translateLocal(errorMessage)}});
```

Comment:
Same here

Reply:
`errorMessage` is also a  phrase key passed as an argument